src/couch/test/couchdb_views_tests.erl

```diff
@@ -543,23 +543,27 @@ has_doc(DocId1, Rows) ->
     lists:any(fun({R}) -> lists:member({<<"id">>, DocId}, R) end, Rows).
 
 backup_db_file(DbName) ->
-    DbDir = config:get("couchdb", "database_dir"),
-    DbFile = filename:join([DbDir, ?b2l(DbName) ++ ".couch"]),
-    {ok, _} = file:copy(DbFile, DbFile ++ ".backup"),
-    ok.
+    {ok, Db} = couch_db:open_int(DbName, []),
```

Comment:
Should we abstract it away into `get_filepath(DbName)`?
In this case it could be used from both `backup_db_file` and `restore_backup_db_files`.